packages/starlight/__tests__/basics/slugs.test.ts

```diff
@@ -95,10 +95,7 @@ describe('urlToSlug', () => {
 		);
 	});
 
-	// It is currently not possible to test this as stubbing BASE_URL is not supported due to
```

Comment:
Note that this test does not need the whole `resetModules()` → `stubEnv()` → `import()` dance as contrary to `base.ts`, `BASE_URL` is not used/evaluated at the top-level scope of the module but in the scope of the function under test.